api-reference/40 Common Types/AIProvider/sendRequest.md

```diff
@@ -0,0 +1,149 @@
+---
+id: AIProvider.sendRequest
+type: function(params)
+---
+---
+##### shortDescription
+A function that sends a request to a language model.
```

Comment:
@marker-dao  is it okay to write "language model" here? I think in the most cases there will be requests to proxy servers

Reply:
```suggestion
A function that sends a request to an AI service
```